partner.py

```diff
@@ -0,0 +1,39 @@
+from local_data_platform import helloworld
+from local_data_platform.entity import User, Product
+from local_data_platform.transaction import Transaction 
+
+
+
+'''
+	Product? Uber 
+	User? Rider & Driver
+	Handshake?
```

Comment:
@mrutunjay-kinagi handshakes need to be setup with python Protocol class

Reply:
@tusharchou , according to the below statement form the Readme:
Local Data Platform
Business information systems require fresh data every day organised in a manner that retrival is cost effective. Making a local data platform requires a setup where you can recreate production usecases and develop new pipelines.

Platforms need to retrieve and store data in a cost effective manner. But in the base class defs, only storage related class defs are seen. Please try to include data retrieval and data conciliation related implementation definition.
As according to the use case we should include real time, as well as concurrent protocols as well

Reply:
### Sample Data

```
business = (product="Uber")
users = [ (id=1, name="Abhisekh Kar", type="Rider") , (id=2, name="Mrutunjay Kinagi", type="Driver")]
listings = [(id=1,user_id=2, cab_no = "XX XX XX XX")
bookings = [(rider_id=1, driver_id=2, listing_id=1, created_at=<ds>, updated_at=<ts>)]
```

Reply:
Demo coming soon

Reply:
Can you plan these demos better